### DISCUSSION TITLE
Option for wayland popup

### DISCUSSION THREAD

**Discussion by CodedBeastVirum** [dynobo/normcap#491] [Q&A]
I noticed that the app saves a screenshot every time i open it to my pictures folder. I did a bit of reading and found that you do this as a work around to the Wayland permission for screenshot dialogue.
I was wondering if it would be possible to make this optional, as it makes a lot of clutter that has to be cleaned up when I use the app.

**Comment by dynobo** [comment]
As far as I know, the fact that Wayland saves the screenshot as a file is independent of wether the screenshot is taken interactively (with a window popping up in between) or non-interactively: In both cases the screenshot is stored. (Please let me know if you have reason to believe otherwise.)

The underlying reason is, that the [org.freedesktop.portal.Screenshot](https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.Screenshot)-spec doesn't support the desired behavior at the moment. E.g. if that API would accept an argument to provide a target folder for the screenshot file, then one could write those files to `/tmp`.

**It would be great, if someone could open and follow up such a feature request in the [xdg-desktop-portal repo](https://github.com/flatpak/xdg-desktop-portal).**

(I am annoyed quite a lot by this behavior myself, not only for NormCap but a lot of other screenshot tools, too. But I don't have enough time to push for this issue upstream.)
